userspace/libsinsp/fields_info.cpp

```diff
@@ -96,7 +96,7 @@ void list_events(sinsp* inspector, bool markdown)
 
 		if(markdown)
 		{
-			if(sinsp::simple_consumer_consider_evtnum(j))
+			if(sinsp::should_consider_evtnum(j))
```

Comment:
again this could be merged with this if:

```c
if((ei.flags & EF_UNUSED) || (ei.flags & EF_OLD_VERSION) || (ei.category & EC_INTERNAL))
{
	continue;
}
```